charts/datadog/ci/agent-otel-collector-no-config-values.yaml

```diff
@@ -0,0 +1,16 @@
+targetSystem: "linux"
+agents:
+  image:
+    repository: datadog/agent-dev
+    tag: nightly-ot-beta-main
+    doNotCheckTag: true
+  containers:
+    agent:
+      env:
+        - name: DD_HOSTNAME
+          value: "datadog"
```

Comment:
is it necessary? 
see my previous comment https://github.com/DataDog/helm-charts/pull/1429#discussion_r1682920108

Reply:
This is for CI Tests.